index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="desktop-769px.css">
+    <link rel="stylesheet" href="animations.css">
+    <title>Essohanam's Portfolio</title>
+</head>
+<body>
+<header>
+    <div class="logo">
+        <a href="#">My Logo</a>
+    </div>
+    <nav class="Navbar">
+        <div class="icon-menu">
+            <div class="vector"></div>
+            <div class="vector"></div>
+            <div class="vector"></div>
+        </div>
+        <div class="links">
+            <ul>
+                <li><a href="#portfolio">Porfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </div>
+    </nav>
+</header>
+<main>
+    <section class="first" id="portfolio">
+        <div class="first-heading">
+            <h1>I'm Essohanam</h1>
+            <h1>Glad to see you!</h1>
+        </div>
+        <div class="h-description">
+            <p>I'm a web developer! I can help you to create a Product or Portfolio website for you. Check out my website and contact me if you find it great.</p> 
+        </div>
+        <div class="t-links">
+            <h3><a href="#">LET'S CONNECT</a></h3>
+            <div class="socials">
+                <ul>
+                    <li><a href="#" class="connect"><img src="images/Twitter.svg" alt="Twitter handle"></a></li>
```

Comment:
According to the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), you should avoid lines of code that are too long (100 characters or more). [check here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Avoid%20lines%20of%20code%20that%20are%20too%20long%20(100%20characters%20or%20more).)